docs/02.API-REFERENCE.md

```diff
@@ -11320,8 +11347,7 @@ backtrace_callback (jerry_backtrace_frame_t *frame_p,
 
 **Summary**
 
-When JERRY_FEATURE_VM_EXEC_STOP is enabled a callback function can be
-specified by this function. This callback is periodically called when
+The callback passed to this function is periodically called when
```

Comment:
How is this change related to the new vm throw? AFAIK these changes should not be in this PR, but in a separate one.

Reply:
I think it is not worth it. Small QOL changes.